spec/CloudCode.spec.js

```diff
@@ -2784,6 +2784,26 @@ describe('afterFind hooks', () => {
     });
   });
 
+  it('should limit count query if limit is set', async done => {
```

Comment:
```suggestion
  it('should limit count query if limit is set', async () => {
```

We're not using `done` anymore.